app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="Translation data from KSU-Electronic Mosshaf project"
+DESCRIPTION="Ayat data is use by other project like elforkane and mtafsir \
+get from original project to minimize maintenance .
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+           2008-2016 Tanzil.net team"
+LICENSE="Tanzil Quran Text"
+REVISION="1"
+SOURCE_URI="http://quran.ksu.edu.sa/ayat/tarajem.ayt"
+CHECKSUM_SHA256="f9c925d86b1cda24f0c706b34c0f7ed3b41464077ffe22ebb37f6d0e279d4f3b"
+SOURCE_DIR="tarajem"
+ARCHITECTURES="any"
+
+PROVIDES="
+	ayat_tarajem_data
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_PREREQUIRES="
+	cmd:find
+	"
+
+BUILD()
+{
+	mkdir -p "$dataDir/ayat/translat"
+	find . -name '*.ayt' -exec sh -c 'mv "$0" "${0%.ayt}.db"' {} \;
+	cp *.db $dataDir/ayat/translat/
+	cat << EOF > $dataDir/ayat/translat/translatInfo
+en_sahih.db,English - Sahih International
+fr_hamidullah.db,Français - Hamidullah
+de_bubenheim.db,Deutsch - Bubenheim & Elyas
+ur_jalandhry.db,أردوUrdu - Fateh Muhammad Jalandhryجالندرب
+ru_kuliev.db,Россию - Кулиев
+sq_nahi.db,Shqiptar - Efendi Nahi
+es_cortes.db,Spanish - Cortes
+bn_bengali.db,Bengali - Muhiuddin Khan
+bs_korkut.db,Bosnian - Korkut
+it_piccardo.db,Italian - Piccardo
+so_abduh.db,Somali - Abduh
+tr_diyanet.db,Turkish - Diyanet Isleri
+ta_tamil.db,Tamil - Jan Turst Foundation
+ha_gumi.db,Hausa - Gumi
+id_indonesian.db,Indonesian - Bahasa Indonesian
+ms_basmeih.db,Malay - Basmeih
+nl_keyzer.db,Dutch - Keyzer
+sw_barwani.db,Swahili - Al-Barwani
+th_thai.db,Thai - ภาษาไทย
+uz_sodik.db,Uzbek - Мухаммад Содик
+zh_jian.db,Chinese - Ma Jian
+pr_tagi.db,فارسى - حسين تاجى كله دارى
```

Comment:
I'm guess there's a font issue on Haiku for these, in Qupzilla they show up as squeres, in Pe/Koder they don't show up at all.

Reply:
> I'm guess there's a font issue on Haiku for these, in Qupzilla they show up as squeres, in Pe/Koder they don't show up at all.

... alter to ascii character all or any other option?

Reply:
I'm guessing it's OK to keep them as is, seeing ayat_tafasir_data has the same issue and is merged already (@korli any thoughts on this?)
@extrowerk anything that needed here?